travis.sh

```diff
@@ -1,5 +1,11 @@
 #!/bin/bash
 
+# Do test
+# cd core/tests
+# env/bin/jasmine-ci --browser phantomjs
+# cd -
```

Comment:
(cd core/tests; env/bin/jasmine-ci --browser phantomjs)

Reply:
Thanks, I take it as a (good) information, but doesn't change it in the PR